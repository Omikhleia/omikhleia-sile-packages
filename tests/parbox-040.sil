\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 040 = From the package documentation, complex case, with footnotes removed.
% I am including it as non-regression as it helped finding a bug with stretch/shrink
% cancellation (medskip is variable).

(4) \parbox[width=37%fw, strut=character, border=0.5pt]{\lorem[words=8]

\center{centered}

\raggedleft{I am ragged\break left}

\medskip

A paragraph after a skip.
}
(…)
\parbox[width=37%fw, strut=character, border=0.5pt]{\lorem[words=16]
} (…)

\end{document}