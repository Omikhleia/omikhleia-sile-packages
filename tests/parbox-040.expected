Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	50.2488
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	23 w=4.6924	(4)
Mx 	37.6251
T	12 w=3.1689	())
Draw line	43.4467	42.0945	198.2268	0.5000
Draw line	43.4467	118.6902	198.2268	0.5000
Draw line	43.4467	42.0945	0.5000	77.0957
Draw line	241.6735	42.0945	0.5000	77.0957
Mx 	43.4467
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	72.2739
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	101.9507
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	128.3560
T	86 76 87 w=10.0146	(sit)
Mx 	142.8472
T	68 80 72 87 w=20.6836	(amet)
Mx 	168.0074
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	216.6442
T	86 68 w=8.4521	(sa)
Mx 	225.0963
T	71 76 83 w=13.2080	(dip)
Mx 	238.3044
T	16 w=3.3691	(-)
Mx 	43.4467
My 	62.2488
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	67.4444
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	124.3863
My 	74.2488
T	70 72 81 87 72 85 72 71 w=36.3477	(centered)
Mx 	192.6627
My 	86.2488
T	44 w=2.9785	(I)
Mx 	198.2838
T	68 80 w=12.6221	(am)
Mx 	213.5485
T	85 68 74 74 72 71 w=28.1250	(ragged)
Mx 	227.7917
My 	98.2488
T	79 72 73 87 w=13.8818	(left)
Mx 	43.4467
My 	116.2488
T	36 w=5.9521	(A)
Mx 	52.0520
T	83 68 85 68 74 85 68 83 75 w=42.6807	(paragraph)
Mx 	97.3857
T	68 73 87 72 85 w=19.7217	(after)
Mx 	119.7605
T	68 w=4.5898	(a)
Mx 	127.0035
T	86 78 76 83 w=16.7480	(skip)
Mx 	143.7515
T	17 w=2.2900	(.)
Mx 	244.3262
My 	50.2488
T	11 w=3.1689	(()
Mx 	247.4951
T	171 w=6.6895	(…)
Mx 	254.1846
T	12 w=3.1689	())
Draw line	260.0062	42.0945	198.2268	0.5000
Draw line	260.0062	76.6902	198.2268	0.5000
Draw line	260.0062	42.0945	0.5000	35.0957
Draw line	458.2329	42.0945	0.5000	35.0957
Mx 	260.0062
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	288.8333
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	318.5101
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	344.9154
T	86 76 87 w=10.0146	(sit)
Mx 	359.4067
T	68 80 72 87 w=20.6836	(amet)
Mx 	384.5669
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	433.2036
T	86 68 w=8.4521	(sa)
Mx 	441.6558
T	71 76 83 w=13.2080	(dip)
Mx 	454.8638
T	16 w=3.3691	(-)
Mx 	260.0062
My 	62.2488
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	284.1125
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	304.3223
T	86 72 71 w=13.6816	(sed)
Mx 	320.7724
T	71 76 68 80 w=20.5322	(diam)
Mx 	344.0731
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	381.1677
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	413.4870
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	446.6363
T	76 81 w=8.2275	(in)
Mx 	454.8638
T	16 w=3.3691	(-)
Mx 	260.0062
My 	74.2488
T	89 76 w=7.6416	(vi)
Mx 	267.6478
T	71 88 81 87 w=19.4580	(dunt)
Mx 	289.7628
T	88 87 w=8.7402	(ut)
Mx 	301.1600
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	460.8856
My 	50.2488
T	11 w=3.1689	(()
Mx 	464.0545
T	171 w=6.6895	(…)
Mx 	470.7440
T	12 w=3.1689	())
End page
Finish
